baggers.js

```diff
@@ -75,6 +75,40 @@ var data = {
         mail: 'benoit.fontaine@live.fr',
       },
     },
+    {
```

Comment:
Could you move this to the end of the list of speakers please?

Reply:
Of course, my bad.